app/views/users/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<!--
+<form action="/users#create" method ="post" accept-charset="UTF-8">
```

Comment:
- [ ] Kindly in the action path, the path should be `users_path` not `/users#create`